### DISCUSSION TITLE
Welcome to blindr Discussions!

### DISCUSSION THREAD

**Discussion by m12t** [m12t/blindr#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jdppujara2010** [comment]
Can you please share the circuit connections I am also trying to add in relays but am not able to work it up.

**Reply by m12t** [reply]
👋 Unfortunately, I don't have the circuit diagram for this. The GPIO pins that I used can be found in the code, though. For the toggle switch, I used GPIO 18, 19, and a ground. [see here](https://github.com/m12t/blindr/blob/main/src/toggle/toggle.c)

Feel free to change around the pinout on your implementation, just make sure the pins used in the code are also updated.
